### PR TITLE
[OSDOCS-11923]: [OSD-GCP] review changes to "Responsibility assignment matrix" page with introduction of GCP PSC 

### PR DESCRIPTION
**Pull request by mletalie**
Version(s):

4.17+
Issue:

Link to docs preview:

https://issues.redhat.com/browse/OSDOCS-11923
QE review:
- [x] QE has approved this change.

https://85503--ocpdocs-pr.netlify.app/openshift-dedicated/latest/osd_architecture/osd_policy/policy-responsibility-matrix.html#incident-operations-management_policy-responsibility-matrix
Additional information:

### CODE REVIEW

File: modules/policy-shared-responsibility.adoc [hunk 1]
```diff
@@ -21,15 +21,17 @@ The customer is responsible for incident and operations management of customer a
...
 |Monitor cloud load balancers and native OpenShift router service, and respond to alerts.
-|* Monitor health of service load balancer endpoints
+|* Monitor health of service load balancer endpoints.
 * Monitor health of application routes, and the endpoints behind them.
 * Report outages to Red Hat.
 
 |Virtual networking
-|Monitor cloud load balancers, subnets, and public cloud components necessary for default platform networking, and respond to alerts.
+|* Monitor cloud load balancers, subnets, and public cloud components necessary for default platform networking, and respond to alerts.
+* Monitor the Red Hat Management project in {GCP} and the Private Service Connect attachments. ^[1]^
```
Comment (jneczypor) [inline modules/policy-shared-responsibility.adoc:30]:
```suggestion
* Monitor the Red Hat Management project in {GCP} and the Private Service Connect (PSC) attachments. ^[1]^
```

File: modules/policy-shared-responsibility.adoc [hunk 2]
```diff
@@ -21,15 +21,17 @@ The customer is responsible for incident and operations management of customer a
...
 |Monitor network traffic that is optionally configured through VPC to VPC connection, VPN connection, or Direct connection for potential issues or security threats.
 
 |===
+1. Applies to {product-title} on {GCP} clusters with Private Service Connect (PSC) only.
```
Comment (jneczypor) [inline modules/policy-shared-responsibility.adoc:34]:
```suggestion
1. Applies to {product-title} on {GCP} clusters with PSC only.
```

File: modules/policy-shared-responsibility.adoc [hunk 3]
```diff
@@ -72,9 +74,11 @@ Red Hat is responsible for enabling changes to the cluster infrastructure and se
 |Virtual networking
 |* Set up and configure virtual networking components required to provision the cluster, including virtual private cloud, subnets, load balancers, internet gateways, NAT gateways, etc.
 * Provide the ability for the customer to manage VPN connectivity with on-premises resources, VPC to VPC connectivity, and Direct connectivity as required through {cluster-manager}.
+* Setup and configure Private Service Connect attachments. ^[1]^
```
Comment (jneczypor) [inline modules/policy-shared-responsibility.adoc:77]:
```suggestion
* Setup and configure PSC attachments. ^[1]^
```

File: modules/policy-shared-responsibility.adoc [hunk 4]
```diff
@@ -72,9 +74,11 @@ Red Hat is responsible for enabling changes to the cluster infrastructure and se
...
 * Enable customers to create and deploy public cloud load balancers for use with service load balancers.
 |* Set up and maintain optional public cloud networking components, such as VPC to VPC connection, VPN connection, or Direct connection.
 * Request and configure any additional service load balancers for specific services.
+* Create a Private Service Connect subnet. ^[1]^
```
Comment (jneczypor) [inline modules/policy-shared-responsibility.adoc:81]:
```suggestion
* Create a PSC subnet. ^[1]^
```

File: modules/policy-shared-responsibility.adoc [hunk 5]
```diff
@@ -93,6 +97,7 @@ Red Hat is responsible for enabling changes to the cluster infrastructure and se
 * Respond to Red Hat notifications regarding cluster resource requirements.
 
 |===
+1. Applies to {product-title} on {GCP} clusters with Private Service Connect (PSC) only.
```
Comment (jneczypor) [inline modules/policy-shared-responsibility.adoc:100]:
```suggestion
1. Applies to {product-title} on {GCP} clusters with PSC only.
```

**Review by jneczypor — COMMENTED** [review]
Just fix the acronyms. We spell out first mention and abbreviate all others. Otherwise, LGTM.
/label peer-review-done

**Comment by mletalie** [comment]
Hello @lento-sun, requesting QE review for this PR. Thanks!

**Pull request merged**